client/post-editor/editor-preview/index.jsx

```diff
@@ -144,7 +144,7 @@ const EditorPreview = React.createClass( {
 						editUrl={ this.props.editUrl }
 						externalUrl={ this.cleanExternalUrl( this.props.externalUrl ) }
 						loadingMessage={ this.props.translate(
-							'{{strong}}One moment, please…{{/strong}} loading your new post.',
+							"{{strong}}One moment, please.{{/strong}} We're loading your preview.",
```

Comment:
Seems a smart move, feels more personal/less robotic.
Also makes sense to be more generic with 'preview' :)